cal-itp/benefits
# Eligibility Start: Contactless Pay Modal

closes #1475 

## What this PR does
- [x] Create Contactless modal
- [x] Add Contactless modal trigger
- [x] Bug: Fixes CSS bug on mobile
- [x] Refactor: Refactor all the modals that are triggered by text (rather than a button or an image, like the first Login.gov modal) - the Eligibility Start/Login.gov modal, Contactless Pay modal and LittlePay modal.
- [x] Fix: Add Go back button to the all informational modals on mobile 

## How to test
1. Desktop: Make sure to open modal by clicking and also, keyboard tab focus + <kbd>enter</kbd>
2. Mobile: Open, view whole modal and close modal
3. Make sure all info modals have Back link and that the link works

## Screenshots

### Mobile: Trigger
<img width="1512" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/e3b3de98-4b6f-4e87-bfbd-a15a948bd9e8">
<img width="1512" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/08a857c6-ab03-40dd-b502-b4ebea1c247e">
<img width="1512" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/7b0d5aa8-f0e7-4dd7-83f6-6648459495d9">
<img width="1512" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/0cf2c444-bd45-49da-80c6-7459c68db4bf">
<img width="1512" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/3b1480eb-d79f-4e81-a546-2929712177d1">
Note: Now that the LittlePay modal is also a text modal, there is a chance that the (?) breaks into a second line, on narrower devices.

### Mobile: Modal

<img width="1512" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/ece5a109-7bdc-4b69-9fcc-6dc908742c82">

### Desktop: Trigger
<img width="1512" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/f5c09fd5-eaf0-4c19-b423-c10c6f020064">
<img width="1110" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/2071619f-1158-44cb-8ed5-c006266765db">

### Desktop: Modal - English
<img width="1512" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/d21b7c80-7786-4fd6-8fbd-9277d5a49108">

### Desktop: Modal - Spanish
<img width="1512" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/da8d8538-0ed7-4553-b07b-c5fb523909d9">

## Mobile: Modal - Eng and Span with Go back button
<img width="1512" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/bf39605c-8268-4ff2-8389-1358592d1fce">
